src/fixturelibrary.ts

```diff
@@ -127,6 +127,11 @@ export class FixtureLibrary {
     return fixture;
   }
 
+  public async getIndex(): Promise<string[]> {
+    const items = await this.fixtureIndex.getIndex();
```

Comment:
I was tempted to use `fixtureCache` here. But I'm not sure if it will be available all the time and with all the data.

Reply:
It defiantly won't.